whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -1391,8 +1391,96 @@
+    },
+
+    "referencedObjectDetails": {
+      "pendingResources": {
+        "_:contribution": {
+          "about": "_:work",
+          "addLink": "contribution",
+          "resourceType": "PrimaryContribution",
+          "embedded": true
+        },
+        "_:agent": {
+          "about": "_:contribution",
+          "link": "agent",
+          "resourceType": "Agent"
+        },
+        "_:workTitle": {
+          "about": "_:work",
+          "addLink": "hasTitle",
+          "resourceType": "Title"
+        },
+        "_:title": {
+          "about": "_:instance",
+          "addLink": "hasTitle", "resourceType": "Title",
+          "embedded": true
+        },
+        "_:describedBy": {
+          "NOTE": "If not possible to link, LinkFinder will change it to a Local identifiedBy property",
+          "about": "_:instance",
+          "addLink": "describedBy", "infer": true,
+          "resourceType": "Record"
+        }
+      },
+      "i1": {
+        "NOTE": "By moving 580 to 500 this indicator will not be usefull anymore",
+        "marcDefault": "0"
+      },
+      "$a": {"about": "_:agent", "property": "label", "infer": true, "TODO:targetMatch": "1xx"},
+      "$b": {"about": "_:instance", "property": "editionStatement", "TODO:targetMatch": "250"},
+      "$c": {"about": "_:workTitle", "property": "qualifier", "NOTE:marc-repeatable": false},
+      "$d": {"about": "_:instance", "property": "provisionActivityStatement", "TODO:targetMatch": "260$c"},
+      "$g": {"about": "_:instance", "addProperty": "part"},
+      "$h": {"about": "_:instance", "addLink": "extent", "resourceType": "Extent", "property": "label", "NOTE:marc-repeatable": false},
+      "$i": {"about": "_:work", "addProperty": "marc:displayText"},
+      "$m": {"about": "_:instance", "property": "marc:materialSpecificDetails"},
+      "$n": {"about": "_:instance", "addLink": "hasNote", "resourceType": "Note", "property": "label"},
+      "$o": {"ignored": true, "NOTE:subfield-count": 0},
+      "$s": {"about": "_:workTitle", "property": "mainTitle", "TODO:targetMatch": "240"},
+      "$t": {"about": "_:title", "property": "mainTitle", "TODO:targetMatch": "222, 245"},
+      "$x": {"about": "_:instance", "addLink": "identifiedBy", "resourceType": "ISSN", "property": "value", "TODO:targetMatch": "022"},
+      "$y": {"about": "_:instance", "addLink": "identifiedBy", "resourceType": "CODEN", "property": "value", "TODO:targetMatch": "030"},
+      "$w": {"aboutNew": "_:describedBy", "property": "controlNumber"},
+      "$6": {"ignored": true, "NOTE": "After inventory of usage, this has been set to ignored"},
+      "$7": {"ignored": true, "NOTE": "After inventory of usage, this has been set to ignored"},
+      "$8": {"ignored": true, "NOTE": "After inventory of usage, this has been set to ignored"},
+      "$9": {"ignored": true, "NOTE": "After inventory of usage, this has been set to ignored"},
+      "subfieldOrder": "6 7 i a s t b c d h g k ... x z w 4 8 9"
```

Comment:
In the subfieldorder there is also a subfield 4? Should this be ignored as well?

Reply:
I haven't done anything yet with subfield 4 since it hasn't been mapped before this refactoring either. But I agree that we need to look into the use of this subfield. But maybe after a merge of this feature?

Reply:
Yes, maybe write a TODO meanwhile so we don't forget 😄